hardware/products/ucm/uc5550.md

```diff
@@ -0,0 +1,257 @@
+# UC5550
+![G400D](images/g400d.jpg)
+# Overview
+---
+The UC5550 is a UCM that supports the STM F4 and F7 processor with 16MB of external RAM. It has a built-on STM WiFi module and an Ethernet PHY. It also has 4MB of external flash for large user programs.
+
+# Specifications
+---
+|					|							|
+| -----------------	| -------------------------	|
+| Processor			| STM 32-bit ARM Cortex-M4	|
+| Blah, blah, blah	| Blah, blah, blah			|
+| Blah, blah, blah	| Blah, blah, blah			|
+| Blah, blah, blah	| Blah, blah, blah			|
+| Blah, blah, blah	| Blah, blah, blah			|
+
+# Resources
+---
+## Documents
+* [Schematic](http://files.ghielectronics.com/downloads/Schematics/Systems/G400D%20Dev%20Board%20Schematic.pdf)
+
+## Design Considerations
+This is where we will put the information that the engineer should be aware in order to make their board.  
+Blah, blah, blah  
+Blah, blah, blah  
+Blah, blah, blah  
+Blah, blah, blah  
+Blah, blah, blah  
+Blah, blah, blah  
+Blah, blah, blah  
+Blah, blah, blah  
+Blah, blah, blah  
+Blah, blah, blah  
+Blah, blah, blah  
+Blah, blah, blah  
+Blah, blah, blah  
+Blah, blah, blah  
+Blah, blah, blah  
+Blah, blah, blah  
+Blah, blah, blah  
+Blah, blah, blah  
+
+## Module Pin Configuration  
```

Comment:
Fix column spacing.

Reply:
Also find out if we can left align the columns.

Reply:
What do you mean fix column spacing? As for the align for columns, they can be aligned left, but there is not an option to have different alignments per each cell unless we use html tags.